src/include/txn.h

```diff
@@ -289,6 +289,9 @@ struct __wt_txn {
     u_int prepare_count;
 #endif
 
+    /* This is a temporary feature flag to simplify resolving uncommitted updates. */
+    bool resolve_uncommitted;
```

Comment:
I think the naming here could get confusing as prepared transaction commits should still resolve updates. But I guess it works for now.

Reply:
Yeah it is temporary - but let me know if a better alternative occurs to you.

Reply:
A little verbose but avoids any confusion: `resolve_weak_hazard_updates` Its whats i had on my PR, thoughts?

Reply:
The functionality in commit/rollback that does the resolution, I have pulled it out in a corresponding sounding function called ```__txn_resolve_uncommitted_op```.

I can change them both to ```resolve_weak_hazard_updates``` and ```__txn_resolve_weak_hazard_updates```

Reply:
Yeah that sounds good to me.